.github/workflows/build-test.yml

```diff
@@ -49,3 +49,39 @@ jobs:
       project_name: ${{ needs.prepare-environment.outputs.project_name }}
       project_version: ${{ needs.prepare-environment.outputs.project_version }}
     secrets: inherit
+
+  benchmark-cpp:
+    name: Benchmark C++
+    runs-on: ubuntu-latest
+    needs:
+      - prepare-environment
+      - build-and-test
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          lfs: true
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Pull Development Image
+        run: docker pull openspacecollective/${{ needs.prepare-environment.outputs.project_name }}-development:${{ needs.prepare-environment.outputs.project_version }}
```

Comment:
I had to hardcode the REGISTRY_OATH (`openspacecollective`) as I was not able to get it from `build-and-test` `env` section.

Any ideas?

Reply:
I think that's reasonable?